debian/conf/nouveau.yaml

```diff
@@ -0,0 +1,28 @@
+maxIndexesOpen: 5000
```

Comment:
how do we get to `5000` here? `max_dbs_open` is `500` by default, should we match that? do we want to go 10x intentionally because we typically expect >1 index per db?

Reply:
there's a multiplier of at least `q` here (would also be a multiplier of `n` if in future you have 1 nouveau server for all N couchdb servers). if you had 500 dbs you might have 500*8 indexes, but really this number is just a guess. I think it will be a debconf parameter and certainly something an admin will change over time.

Reply:
our default `q` is 2 and `n` would be `3`, so we’d be at `500 * 3 * 2 = 3000` — I don’t mind adding 2k for good measure, but it might be nice to choose a default that has a justification that allows admins to change this when they deviate from our defaults in other places, sort of teaching how this all fits together.

Reply:
pick a number? There's an LRU (hopefully better than the one I wrote for clouseau) so the number here is a guess at how many we think a server could have open and active at once. 5000 is probably high then. 

1000?

Reply:
I thought max_open_files * q * n defaults was a good measure: 3000

Reply:
deal